jme3-alloc/src/main/java/com/jme3/alloc/util/NativeErrno.java

```diff
@@ -42,12 +41,7 @@
 public final class NativeErrno {
 
     static {
-        try {
-            /* extracts and loads the system specific library if it's not been loaded before */
-            NativeBinaryLoader.loadLibrary();
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
+        NativeBinaryLoader.quickLoadLibrary();
     }
```

Comment:
Why do you load the library here? Should not it be enough to load it in NativeBufferUtils?

Reply:
That was placed to extract (if not extracted) and load the library if someone utilizes NativeErrno before the BufferUtils use.
But, technically, they shouldn't.

Reply:
Btw, the System#load(String) subsequent calls are ignored.

Reply:
For extra safety, further guarding can be added after working on https://github.com/Software-Hardware-Codesign/jme-alloc/issues/62.